pandas/tests/computation/test_eval.py

```diff
@@ -411,7 +411,7 @@ def check_single_invert_op(self, lhs, cmp1, rhs):
                 )
 
     def check_compound_invert_op(self, lhs, cmp1, rhs):
-        skip_these = "in", "not in"
+        skip_these = ["in", "not in"]
```

Comment:
Maybe make a tuple for consistency with old code?

Reply:
Just used for an inclusion check; either way works.